charts/kof-regional/templates/regional-multi-cluster-service.yaml

```
@@ -110,3 +110,10 @@ spec:
                 defaultClusterId: %q
           ` .Cluster.metadata.name .Cluster.metadata.name | fromYaml {{`}}`}}
           {{`{{ mergeOverwrite $collectorsValuesHere $collectorsValuesFromHelm $collectorsValuesFromAnnotation | toYaml | nindent 4 }}`}}
+    templateResourceRefs:
+    - resource:
+        apiVersion: cluster.x-k8s.io/v1beta1
+        kind: Cluster
+        name: "{{`{{ .Cluster.metadata.name }}`}}"
+        namespace: "{{`{{ .Cluster.metadata.namespace }}`}}"
+      identifier: MyCluster
```

Please add `optional: true`, otherwise it doesn't work on adopted clusters

```suggestion
    templateResourceRefs:
    - resource:
        apiVersion: cluster.x-k8s.io/v1beta1
        kind: Cluster
        name: "{{`{{ .Cluster.metadata.name }}`}}"
        namespace: "{{`{{ .Cluster.metadata.namespace }}`}}"
      identifier: RegionalCluster
      optional: true
```